tests/python_client/testcases/test_query.py

```diff
@@ -1279,12 +1282,26 @@ def test_query_string_expr_with_binary(self):
     def test_query_string_expr_with_prefixes(self):
         """
         target: test query with 
-        method: specify string primary field as output field
-        expected: return string primary field
+        method: specify string is primary field, use prefix string expr
+        expected: verify query successfully
         """
         collection_w, vectors = self.init_collection_general(prefix, insert_data=True, primary_field=ct.default_string_field_name)[0:2]
         res = vectors[0].iloc[:1, :3].to_dict('records')
         expression = 'varchar like "0%"'
         output_fields = [default_int_field_name, default_float_field_name, default_string_field_name]
         collection_w.query(expression, output_fields=output_fields, 
                            check_task=CheckTasks.check_query_results, check_items={exp_res: res})
+    
+    @pytest.mark.tags(CaseLabel.L1)
+    def test_query_string_with_invaild_prefix_expr(self):
+        """
+        target: test query with 
+        method: specify string primary field, use invaild prefix string expr
+        expected: raise error
+        """
+        collection_w, vectors = self.init_collection_general(prefix, insert_data=True)[0:2]
```

Comment:
I think this should be better: "collection_w = self.init_collection_general(prefix, insert_data=True)[0]"

Reply:
Thank you, already modified